build.gradle

```diff
@@ -222,12 +223,4 @@ class Checksum {
     private static toHex(int b) {
         return hexChr((b & 0xF0) >> 4) + hexChr(b & 0x0F)
     }
-
-    static def main(args) {
-        def sha1sum = Checksum.getSha1sum(new File(args[0]))
-        println "file($args[0]): sha1sum($sha1sum)"
-    }
-
```

Comment:
Any reason why we deleted this?

Reply:
It didn't seem like it was being used anywhere in the code, so I deleted it. To be certain, I did a search on TeamCity build logs.